src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -12219,56 +12219,56 @@ void writeVeriSumSpaceTables(EnergyPlusData &state, bool produceTabular, bool pr
 
     // Accumulate internal gain totals by space and space type
     for (int iPeople = 1; iPeople <= state.dataHeatBal->TotPeople; ++iPeople) {
-        auto curPeople = state.dataHeatBal->People(iPeople);
+        auto &curPeople = state.dataHeatBal->People(iPeople);
         int const spaceNum = curPeople.spaceIndex;
```

Comment:
Here and at a few occurrences that follow, reference type `auto &` should be used instead of plain `auto` to avoid copy of large struct objects.